src/components/SearchOverlay.vue

```diff
@@ -200,6 +206,12 @@ function searchFor(term: string) {
                                 Showing results for <strong>{{ filters.term }}</strong>
                             </h2> 
                             <span v-if="result.hits > 0">Showing {{ page * 30 - 29 }} - {{ result?.hits < 30 ? result?.hits : page * 30 }} of {{ result?.hits }}</span>
+                            <div class="hidden lg:block lg:flex-grow">
+                            </div>
+                            <select v-model="filters.sort" class="text-sm lg:text-base w-full lg:w-1/6" @change="search">
+                                <option>Relevance</option>
+                                <option>Popular</option>
```

Comment:
Please use same sorting options as on `Category`

Reply:
The price sorting? 
Currently these two options are now the same.
The category page has sorting on prices asc and decs as well.
That makes no sense to do here.

Reply:
Why does that make no sense here?

And yes, I want the same 4 options both places :)
(Could extract helper functions and components to make keeping them in sync easier)

Reply:
I have extracted it to it's own component! 😊